### ISSUE TITLE
Error: ENOENT: no such file or directory, scandir ''

### ISSUE THREAD

**Issue by rafw87** [log4js-node/streamroller#23]
Full error:
```
Error: ENOENT: no such file or directory, scandir ''
    at Error (native)
    at Object.fs.readdirSync (fs.js:951:18)
    at RollingFileStream._getExistingFiles (D:\GIT\log4js-test\node_modules\streamroller\lib\RollingFileWriteStream.js:247:52)
    at RollingFileStream._initState (D:\GIT\log4js-test\node_modules\streamroller\lib\RollingFileWriteStream.js:119:38)
    at RollingFileStream.RollingFileWriteStream (D:\GIT\log4js-test\node_modules\streamroller\lib\RollingFileWriteStream.js:67:11)
    at new RollingFileStream (D:\GIT\log4js-test\node_modules\streamroller\lib\RollingFileStream.js:33:124)
    at Object.<anonymous> (D:\GIT\log4js-test\streamroller_test.js:2:14)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
```

Reproduceable on sample code from README:
```js
var rollers = require('streamroller');
var stream = new rollers.RollingFileStream('myfile', 1024, 3);
stream.write("stuff");
stream.end();
```

Environment:
- Node: v6.9.1
- NPM: 3.10.8
- **Windows 7**

Packages tree:
```
`-- streamroller@0.8.3
  +-- debug@3.1.0
  | `-- ms@2.0.0
  +-- fs-extra@5.0.0
  | +-- graceful-fs@4.1.11
  | +-- jsonfile@4.0.0
  | `-- universalify@0.1.1
  +-- lodash@4.17.10
  `-- moment@2.22.1
```

**Comment by nomiddlename** [comment]
I've seen this issue in the log4js tests as well. I'll fix it.

**Comment by nomiddlename** [comment]
Fix published to npm in version 0.8.4
